apple_server.js

```diff
@@ -67,8 +67,9 @@ const verifyAndParseIdentityToken = (query, idToken, isNative = false) =>
  * @param query
  * @param {*} tokens tokens and data from apple
  * @param isNative
+ * @param fromLoginHandler
  */
-const getServiceDataFromTokens = (query, tokens, isNative = false) => {
+const getServiceDataFromTokens = (query, tokens, isNative = false, fromLoginHandler = false) => {
```

Comment:
this name `fromLoginHandler` should be more descriptive.

```suggestion
const getServiceDataFromTokens = (query, tokens, isNative = false, isBeingCalledFromLoginHandler = false) => {
```